lib/common.bash

```diff
@@ -171,6 +171,16 @@ clean_env()
 	fi
 }
 
+clean_env_ctr()
+{
+	check_containers=$(sudo ctr c list -q | wc -l)
+	if ((${present})); then
```

Comment:
This doesn't look right to me as you've created a variable, but are then checking an non-existent one?